deployment/helm/human-connection/templates/service-backend.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name:  {{ .Release.Name }}-backend
+  namespace: human-connection
```

Comment:
Are you sure? Consider that you could run two different instances of human connection in the same kubernetes cluster in two different namespaces. As far as I'm concerned, this is the reason why setting the namespace is considered bad practice in helm charts. You can set it when you install the chart.

Reply:
You are right! I still have some cleanup to do, thanks for pointing this out!